kqueen/engines/jenkins.py

```diff
@@ -118,6 +151,17 @@ def provision(self, **kwargs):
         # PATCH THE CTX TO CONTAIN CLUSTER NAME AND UUID
         ctx[cluster_name] = 'kqueen-{}'.format(self.cluster.id)
         ctx[cluster_uuid] = self.cluster.id
+        if self.job_override_params:
+            for key, value in self.job_override_params:
+                ctx[key] = ctx[value]
+
+        for param in ctx.keys():
+            if not self._parameter_exist(param):
+                msg = 'Failed to manage job parameter: {}'.format(param)
+                logger.error(msg)
+                return False, msg
+
+        logger.debug('Current Job configuration: {}, provisioning started...'.format(ctx))
```

Comment:
actually, it is started after 166 line

Reply:
there are try catch block, so i prefer to log here, its bad?

Reply:
it's better to place it inside try-catch block